torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -207,13 +207,23 @@ def convert_color_space_image_pil(
     return image.convert(new_mode)
 
 
+def convert_color_space_video(
+    video: torch.Tensor, old_color_space: ColorSpace, new_color_space: ColorSpace, copy: bool = True
+) -> torch.Tensor:
+    return convert_color_space_image_tensor(
+        video, old_color_space=old_color_space, new_color_space=new_color_space, copy=copy
+    )
+
+
 def convert_color_space(
-    inpt: features.ImageTypeJIT,
+    inpt: features.ImageOrVideoTypeJIT,
     color_space: ColorSpace,
     old_color_space: Optional[ColorSpace] = None,
     copy: bool = True,
-) -> features.ImageTypeJIT:
-    if isinstance(inpt, torch.Tensor) and (torch.jit.is_scripting() or not isinstance(inpt, features.Image)):
+) -> features.ImageOrVideoTypeJIT:
+    if isinstance(inpt, torch.Tensor) and (
+        torch.jit.is_scripting() or not isinstance(inpt, (features.Image, features.Video))
```

Comment:
This specific `isinstance` check for image or video is correct, but I think we missed it in all other places on the dispatchers. I think we need to add this everywhere.

Reply:
I went through the whole API again and found a few places where we need to decide if we want video support there:

- classificaton transforms: `MixUp`, `CutMix`
- object detection transforms: `SimpleCopyPaste`, `RandomIoUCrop`
- deprecated transforms: `Grayscale`, `RandomGrayscale`
- outlier transforms: `FiveCrop`, `TenCrop` (we are not supporting anything besides images here so far)
- image only transforms (by name): `ConvertImageDtype`

For everything else this is fixed.